sklearn/model_selection/tests/test_validation.py

```diff
@@ -1450,3 +1450,12 @@ def test_fit_and_score():
     # check if the same warning is triggered
     assert_warns_message(FitFailedWarning, warning_message, _fit_and_score,
                          *fit_and_score_args, **fit_and_score_kwargs)
+
+    # check if exception was raised, with default error_score argument
+    assert_raise_message(ValueError, "", _fit_and_score,
```

Comment:
Why do we not test for the message text?

Reply:
Can you please elaborate on this? Maybe I'm wrong, but when raise method is invoked there is no message.

Reply:
It should be the original error from `fit`, in this case "Failing classifier failed as required"